### PR TITLE
[SWE-1424][BRANDBOOK] Write main color component

### PR DESCRIPTION
**Pull request by ArtemSuhoy**
image

### CODE REVIEW

File: src/app/domains/Color/components/ColorSimpleView/СolorSimpleView.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          </Box>
+          <Box>
+            <Text variant="body1" color={typographyColor}>
+              {rgb}
+            </Text>
+          </Box>
+        </Col>
+      </Row>
+    </ColorSimpleViewStyled>
+  )
```
Comment (olenashevchuk) [inline src/app/domains/Color/components/ColorSimpleView/СolorSimpleView.js:35]: don't use grid there

File: src/app/domains/Color/components/ColorSimpleView/СolorSimpleView.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <ColorSimpleViewStyled variant={variant}>
+      <Row>
+        <Col cw={12} pb={16}>
+          <Title variant="heading6" color={typographyColor}>
+            {title}
+          </Title>
+        </Col>
+        <Col cw={12}>
+          <Box>
+            <Text variant="body1" color={typographyColor}>
```
Comment (olenashevchuk) [inline src/app/domains/Color/components/ColorSimpleView/СolorSimpleView.js:23]: !!!

File: src/app/domains/Color/components/ColorSimpleView/СolorSimpleView.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import React from 'react'
+
+const ColorSimpleView = (props) => {
+  const { variant, title, hex, rgb } = props
+
+  // [COMPUTED_PROPERTIES]
+  const typographyColor = variant === 'dark' && 'white'
+
+  return (
+    <ColorSimpleViewStyled variant={variant}>
```
Comment (olenashevchuk) [inline src/app/domains/Color/components/ColorSimpleView/СolorSimpleView.js:14]: use color instead variant prop

File: src/app/domains/Color/components/ColorAdvancedView/ColorAdvancedView.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import { Box, Col, Row } from 'components'
+
+import { ColorSimpleView } from 'domains/Color/components'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const ColorAdvancedView = (props) => {
```
Comment (olenashevchuk) [inline src/app/domains/Color/components/ColorAdvancedView/ColorAdvancedView.js:7]: it is not ColorAdvancedView , it is ColorsList maybe

**Review by olenashevchuk — APPROVED** [review]

File: src/app/components/bundles/ContentScreen/ContentScreen.js [hunk 5]
```diff
@@ -2,8 +2,8 @@ import {
   ContentScreenContainer,
   ContentScreenContent,
   ContentScreenInfo
-} from 'components/bundles/ContentScreen/components'
-import { Footer, Header } from 'components'
+} from './components'
+
 
 import ContentScreenStyled from './ContentScreen.styled'
 import React from 'react'
```
Comment (Olexandr25) [inline src/app/components/bundles/ContentScreen/ContentScreen.js:9]: do you need React for this component?

File: src/app/domains/Color/components/ColorList/ColorList.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import { Box, Col, Row } from 'components'
+
+import { ColorSimpleView } from 'domains/Color/components'
+import PropTypes from 'prop-types'
+import React from 'react'
```
Comment (Olexandr25) [inline src/app/domains/Color/components/ColorList/ColorList.js:5]: do you need this React here?

File: src/app/domains/Color/components/ColorList/ColorList.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+const ColorList = (props) => {
```
Comment (Olexandr25) [inline src/app/domains/Color/components/ColorList/ColorList.js:7]: write comment - js doc

File: src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.js [hunk 8]
```diff
@@ -0,0 +1,39 @@
+import { Box, Text, Title } from 'components'
+
+import ColorSimpleViewStyled from './ColorSimpleView.styled.js'
+import PropTypes from 'prop-types'
+import React from 'react'
```
Comment (Olexandr25) [inline src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.js:5]: do you need React here?

File: src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.js [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+const ColorSimpleView = (props) => {
```
Comment (Olexandr25) [inline src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.js:7]: comment js doc

File: src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.styled.js [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+    background-color: var(--brandbook-module-white);
+    border: 1px solid rgba(0, 0, 0, 0.16);
+  `,
+  dark: css`
+    background-color: var(--brandbook-module-neutral);
+  `
+}
+
+const ColorSimpleViewStyled = styled.div`
+  padding: 32px 40px;
```
Comment (whoiam460) [inline src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.styled.js:17]: move it to vars

**Pull request closed without merging**